plugins/modules/ntnx_vms.py

```diff
@@ -680,27 +729,22 @@
   sample: "82c5c1d3-eb6a-406a-8f58-306028099d21"
 """
 
-from ..module_utils.base_module import BaseModule  # noqa: E402
+
+from ..module_utils import utils  # noqa: E402
 from ..module_utils.prism.tasks import Task  # noqa: E402
+from ..module_utils.prism.vm_base_module import VMBaseModule  # noqa: E402
 from ..module_utils.prism.vms import VM  # noqa: E402
-from ..module_utils.utils import remove_param_with_none_value  # noqa: E402
 
 
 def get_module_spec():
     mutually_exclusive = [("name", "uuid")]
 
     entity_by_spec = dict(name=dict(type="str"), uuid=dict(type="str"))
 
-    network_spec = dict(
```

Comment:
There are some params like network_spec, vcpus, etc are removed. Please check and restore them.